### PR TITLE
Add comment to explain arkade full path usage in install script

### PR DESCRIPTION
**Pull request by welteki**
Signed-off-by: Han Verstraete (OpenFaaS Ltd) <han@openfaas.com>

## Description

Add comment to explain arkade full path usage in install script.

## Motivation and Context

- [ ] I have raised an issue to propose this change **this is required**
Requested in: https://github.com/openfaas/faasd/pull/276#issuecomment-1141050493

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Commits:

- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] My commit message has a body and describe how this was tested and why it is required.
- [x] I have signed-off my commits with `git commit -s` for the Developer Certificate of Origin (DCO)

Code:

- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.

Docs:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by alexellis — APPROVED** [review]
Approved

File: hack/install.sh [hunk 1]
```diff
@@ -67,6 +67,9 @@ install_arkade(){
   arkade --help
 }
 
+# On CentOS /usr/local/bin is not included in the PATH when using sudo.
```
Comment (alexellis) [inline hack/install.sh:70]: I expected to see this on the declaration of the `ARKADE` environment variable, since it may be used multiple times.

Comment (welteki) [reply hack/install.sh]: I moved the comment

**Review by alexellis — APPROVED** [review]
LGTM

**Pull request merged**